sos/report/plugins/postgresql.py

```diff
@@ -19,7 +19,7 @@
 from sos.utilities import find
 
 
-class PostgreSQL(Plugin):
+class PostgreSQL(Plugin, SCLPlugin):
```

Comment:
`convert_cmd_scl` from SCLPlugin is used here.

Reply:
@pmoravec do we still need to support postgres as an SCL? This rings a bell to me as something we talked about dropping before.

Reply:
Agree, let drop it. SCL is RHEL7 based, which is legacy-3.9 only.

In fact, we can imho drop whole SCL stuff like [`add_copy_spec_scl`](https://github.com/sosreport/sos/blob/main/sos/report/plugins/__init__.py#L3596)which is quite indepentent task that deserves [own (discussion and?) issue](https://github.com/sosreport/sos/issues/3559).

I am ok to either drop it here now or within the work for the #3559.

Reply:
I'd say drop it later, so it's all contained. We can leave it here as this PR is focused on linting and not code flow.